docs/hu/docs/index.md

```diff
@@ -0,0 +1,463 @@
+* Ad egy `PUT` kérést a `/items/{item_id}`-hez, beolvassa a body-t JSON-ként:
+    * Leellenőrzi a kötelező `name` mezőt, hogy az `str`.
+    * A kötelező `price` mezőnek pedig `float`-nak kell lennie.
+    * Azt is megvizsgálja, hogy az opcionális `is_offer` mező jelen van-e és `bool`-e.
+    * És mindez a mélyen beágyazott JSON objektumok esetén is működik.
+* Automatikusan konvertál JSON-ba és JSON-ból.
+* Mindent dokumentál az OpenAI-val, amit lehet használni:
+    * Interaktív dokumentációs rendszerekben
+    * Automata kliens kód generáló rendszerekben, sokféle nyelvhez illesztve.
+* Közvetlenül kétféle interaktív webes dokumentációs rendszert biztosít.
+
+---
+
+Csak a felszínt karcoljuk meg, és máris jobban érted, hogyan működik az egész.
+
+Próbáld ki, hogy megváltoztatod ebben a sorban:
+
+```Python
+    return {"item_name": item.name, "item_id": item_id}
+```
+
+...ezt:
+
+```Python
+        ... "item_name": item.name ...
+```
+
+...erre:
+
+```Python
+        ... "item_price": item.price ...
+```
+
+...és látni fogod, hogy a szerkesztő automatikuasn kiegészíti a mezőt és ismeri a típusát is:
+
+![editor support](https://fastapi.tiangolo.com/img/vscode-completion.png)
+
+Ha egy részletesebb példát szeretnél látni, amelyik még több funkciót mutat be, nézd meg az <a href="https://fastapi.tiangolo.com/tutorial/">Útmutató - Felhasználói kézikönyvet</a>.
+
+**Spoiler alert**: az útmutató - felhasználói kézikönyv tartalmaz:
+
+* **Paraméterek** deklarációját különféle helyeken: **header**, **cookie**, **form mező** és **fájl**.
+* Hogyan állíts be **ellenőrzési megszorításokat**, mint pl. a `maximum_length` vagy a `regex`.
+* Hogyan használd a nagyon hatékony és könnyű **<abbr title="más néven komponensek, erőforrások (resources), szolgáltatók (providers), szervízek (services), befecskendezések (injections)">Dependency Injection (Függőség befecskendezése)</abbr>** rendszert.
+* Biztonság és autentikáció, **OAuth2** **JWT tokenekkel** és **HTTP Basic** auth támogatása.
+* Fejlettebb (de hasonlóan egyszerű) megoldások **mélyen beágyazott JSON modellek** deklarálására (a Pydantic-nak köszönhetően).
+* **GraphQL** integráció a <a href="https://strawberry.rocks" class="external-link" target="_blank">Strawberry</a>-vel és egyéb könyvtárakkal.
+* Rengeteg extra képesség (a Starlette-nek köszönhetően), mint pl.:
+    * **WebSocketek**
+    * nagyon egyszerű tesztek HTTPX-en és `pytest`-en
+    * **CORS**
+    * **Cookie Session-ök**
+    * ...és még sok minden más.
+
+## Teljesítmény
+
+Uvicorn alatt futtatott **FastAPI** alkalmazások független TechEmpower teljesítménymutatóit láthatod a <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">one of the fastest Python frameworks available</a> linken, csak Starlette és Uvicorn alatt (a FastAPI beépítetten használja). (*)
```

Comment:
"az egyik leggyorsabb elérhető Python keretrendszer" instead of "one of the fastest Python frameworks available"